fix(evpn): unit test fixes

atulpatel261194 opened the pull request.

artek-koltun commented on network/logical_bridge_test.go:
```diff
@@ -3,161 +3,220 @@
 import (
 	"context"
-	"errors"
+	// "errors"
```
we can just remove the package if it is not used anymore

artek-koltun replied:
can be removed

artek-koltun commented on network/logical_bridge_test.go:
```diff
@@ -3,161 +3,220 @@
+		Name:         "//network.opiproject.org/bridges/lb1",
+		AllowMissing: true,
+	}
+	wantResponse := &emptypb.Empty{}
```
we don't really need to test expected response for `Empty` type. It is just an empty message, which cannot be used in any way

artek-koltun commented on network/vrf_test.go:
```diff
@@ -3,160 +3,383 @@
+	pb "github.com/opiproject/opi-api/network/evpn-gw/v1alpha1/gen/go"
+	"google.golang.org/grpc"
+
+	// pc "github.com/opiproject/opi-api/network/opinetcommon/v1alpha1/gen/go"
```
this line can be removed

artek-koltun reviewed (commented):
@atulpatel261194 a great improvements of the tests! Nice to see +17.80% of the overall coverage. However, there are some comments from my side.

atulpatel261194 commented on network/bridge_port_test.go:
```diff
@@ -3,162 +3,392 @@
+// TestCreateBridgePortError tests the CreateBridgePort method
 func TestCreateBridgePortError(t *testing.T) {
-	mockEvpnClient := &mocks.EvpnClient{}
+	wantRequest := &pb.CreateBridgePortRequest{
```
@artek-koltun I appreciate your feedback, I want to acknowledge that I am currently on leave for the next two weeks. However, please be assured that I am committed to addressing the suggestions you provided as soon as I return in January'24 . I have made note of your suggestions and will prioritize implementing the necessary changes to improve the quality and effectivenes.

atulpatel261194 commented:
> @atulpatel261194 a great improvements of the tests! Nice to see +17.80% of the overall coverage. However, there are some comments from my side.

I appreciate your feedback, I want to acknowledge that I am currently on leave for the next two weeks. However, please be assured that I am committed to addressing the suggestions you provided as soon as I return in January'24 . I have made note of your suggestions and will prioritize implementing the necessary changes to improve the quality and effectivenes

artek-koltun commented:
> I appreciate your feedback, I want to acknowledge that I am currently on leave for the next two weeks. However, please be assured that I am committed to addressing the suggestions you provided as soon as I return in January'24 . I have made note of your suggestions and will prioritize implementing the necessary changes to improve the quality and effectivenes

Sure. Enjoy your vacation!

artek-koltun commented on network/svi.go:
```diff
@@ -122,12 +122,9 @@ func (c evpnClientImpl) UpdateSvi(ctx context.Context, name string, updateMask [
 	defer closer()
 	client := c.getEvpnSVIClient(conn)
-	svi, err := client.GetSvi(ctx, &pb.GetSviRequest{
+
```
can we extract that change in `svi.go` file into a separate commit like `fix(network/svi): create svi directly for update`

artek-koltun reviewed (commented):
Looks good. Just some minor comments

artek-koltun reviewed (approved):
LGTM!
